README.md

```diff
@@ -40,33 +40,25 @@ All of the loops support these features:
 
 Here is an async HTTP server built with just the event loop.
 ```php
-    $loop = React\EventLoop\Factory::create();
-
     $server = stream_socket_server('tcp://127.0.0.1:8080');
     stream_set_blocking($server, 0);
-    $loop->addReadStream($server, function ($server) use ($loop) {
+    React\EventLoop\addReadStream($server, function ($server) {
         $conn = stream_socket_accept($server);
         $data = "HTTP/1.1 200 OK\r\nContent-Length: 3\r\n\r\nHi\n";
-        $loop->addWriteStream($conn, function ($conn) use (&$data, $loop) {
+        React\EventLoop\addWriteStream($conn, function ($conn) use (&$data) {
             $written = fwrite($conn, $data);
             if ($written === strlen($data)) {
                 fclose($conn);
-                $loop->removeStream($conn);
+                React\EventLoop\removeStream($conn);
             } else {
                 $data = substr($data, $written);
             }
         });
     });
 
-    $loop->addPeriodicTimer(5, function () {
+    React\EventLoop\addPeriodicTimer(5, function () {
         $memory = memory_get_usage() / 1024;
         $formatted = number_format($memory, 3).'K';
         echo "Current memory usage: {$formatted}\n";
     });
-
-    $loop->run();
 ```
-**Note:** The factory is just for convenience. It tries to pick the best
-available implementation. Libraries `SHOULD` allow the user to inject an
-instance of the loop. They `MAY` use the factory when the user did not supply
-a loop.
```

Comment:
What i meant was: i surely add docs, but only after the other reviewers have not rejected this RFC as pure non-sense :P